src/sagemaker/fw_utils.py

```diff
@@ -71,6 +71,7 @@
     "pytorch-serving",
 ]
 PY2_RESTRICTED_EIA_FRAMEWORKS = ["pytorch-serving"]
+PY37_SUPPORTED_FRAMEWORKS = ["tensorflow-scriptmode"]
```

Comment:
why do we need this? should we just return image not found if the given combination (Framework, Framework version, Python version) is not support?

Reply:
If we strip this line (L74) Python SDK will have to hit ECR service to find out if the combination exists.

If we do nothing to check for the py37 support before constructing the image uri, [here](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/fw_utils.py#L105) will throw ``KeyError`` (key "py37" not found).

Updating the ``MERGED_FRAMEWORKS_LOWEST_VERSIONS`` dict to let each entry have a "py_version" key is also doable to prevent Python SDK hit the ECR service, but I think this dict is for checking whether the image is DLC merged frameworks instead of checking Python version support.